manifests/server/installdb.pp

```diff
@@ -19,7 +19,7 @@
       creates   => "${datadir}/mysql",
       logoutput => on_failure,
       path      => '/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin',
-      require   => Package[$mysql::server::package_name],
+      require   => Package['mysql-server'],
```

Comment:
As it should fail if the server package is not installed. Perhaps a rework of required with a 
```
notify{'mysql-server': }
...
package{$mysql::package_name:
before => Notify['mysql-server']
}
```
this way others can subscribe to the class and have ordering